src/__fixtures__/state.tsx

```diff
@@ -0,0 +1,78 @@
+import type {
+  Comment,
+  CommentsState,
+  CommentMode,
+  CommentReply,
+  CommentReplyMode,
+} from '../state/comments';
+
+export const basicCommentsState: CommentsState = {
+  focusedComment: 1,
+  pinnedComment: 1,
+  remoteCommentCount: 1,
+  comments: new Map([
+    [
+      1,
+      {
+        contentpath: 'test_contentpath',
+        localId: 1,
+        annotation: null,
+        remoteId: 1,
+        mode: 'default' as CommentMode,
+        deleted: false,
+        author: { id: 1, name: 'test user' },
+        date: 0,
+        text: 'test text',
+        newReply: '',
+        editPreviousText: '',
+        remoteReplyCount: 1,
+        replies: new Map([
+          [
+            2,
+            {
+              localId: 2,
+              remoteId: 2,
+              mode: 'default' as CommentReplyMode,
+              author: { id: 1, name: 'test user' },
+              date: 0,
+              text: 'a reply',
+              editPreviousText: '',
+              deleted: false,
+            } as CommentReply,
+          ],
+          [
+            3,
+            {
+              localId: 3,
+              remoteId: null,
+              mode: 'default' as CommentReplyMode,
+              author: { id: 1, name: 'test user' },
+              date: 0,
+              text: 'another reply',
+              editPreviousText: '',
+              deleted: false,
+            } as CommentReply,
```

Comment:
**Suggestion:** There way too much casting happening here. It looks a little dodgy on the first look. I never have to do it unless I need to cast dynamic data (e.g. API response) for static type checking purposes. Are your objects missing some properties or have type mismatch? I don't think you should be casting at all. Especially as this is just for static type checking - at runtime they are just JS objects.

Reply:
I've removed most of it now, thanks. Was getting a lot of 'implicitly has 'any' type' for the nullable `annotation` and `remoteId`, which was why I added, but adding explicit types to the comment variables in the first place seems to fix it.